htdocs/admin/multicurrency.php

```diff
@@ -41,10 +42,17 @@
 
 // Parameters
 $action = GETPOST('action', 'alpha');
-
+$sync_response = GETPOST('sync_response');
 /*
  * Actions
  */
+if (!empty($sync_response))
+{
+	$sync_response = json_decode($sync_response);
+	MultiCurrency::syncRates($sync_response);
+	exit;
```

Comment:
The exit is still wrong here.
This means the code behind MultiCurrency::syncRates($sync_response); is just a code to run an upate of database from javascript main page. If yes, you must create another dedicate page called   ajax/updaterates.php for example to receive the parameter "sync_response" and make the silent update.